stac_fastapi/types/stac_fastapi/types/transaction.py

```diff
@@ -0,0 +1,125 @@
+"""Transaction extension types."""
+
+import sys
+from typing import Any, Dict, List, Literal, Optional, Union
+
+import attr
+from fastapi import Body
+from pydantic import BaseModel
+from stac_pydantic import Collection, Item, ItemCollection
+from stac_pydantic.shared import BBox
+from typing_extensions import Annotated
+
+from stac_fastapi.api.models import CollectionUri, ItemUri
+
+if sys.version_info < (3, 12, 0):
+    from typing_extensions import TypedDict
+else:
+    from typing import TypedDict
+
```

Comment:
I'm pretty sure we can simply do `from typing_extensions import TypedDict` 

typing extensions will use the native TypedDict is available

Reply:
I followed the import in [types/stac.py](https://github.com/stac-utils/stac-fastapi/blob/main/stac_fastapi/types/stac_fastapi/types/stac.py#L8-L15). Should I update both to `from typing_extensions import TypedDict`?

Reply:
yes please 🙏